README.md

```diff
@@ -94,6 +94,30 @@ Raise an exception if not successful with `run!`:
 Scmd.new("cd /path/that/does/not/exist").run! #=> Scmd::Command::Failure
 ```
 
+### Environment variables
+
+Pass environment variables:
+
+```ruby
+cmd = Scmd.new("echo $TEST_VAR", {
+  :env => 'TEST_VAR' => 'hi'
+})
+```
+
+### Process spawn options
+
+Pass options:
+
+```ruby
+reader, writer = IO.pipe
```

Comment:
@jcredding I'd add a comment here saying "this is an example that uses file descriptor redirection options" to give context to the code below.